lib/matplotlib/markers.py

```diff
@@ -213,12 +214,23 @@ def __init__(self, marker=None, fillstyle=None):
 
         Parameters
         ----------
-        marker : str or array-like, optional, default: None
+        marker : str, array-like, `~.path.Path`, or `~.markers.MarkerStyle`, \
+        default: None
             See the descriptions of possible markers in the module docstring.
 
         fillstyle : str, optional, default: 'full'
             'full', 'left", 'right', 'bottom', 'top', 'none'
+
+        normalization : str, {'classic', 'none'}, optional, default: "classic"
+            The normalization of the marker size. Only applies to custom paths
+            that are provided as array of vertices or `~.path.Path`.
+            Can take two values:
+            *'classic'*, being the default, makes sure the marker path is
```

Comment:
`classic` is a confusing name, because it overlaps with the classic style, but it's _not_ the classic style (I mean it is _also_ the classic style, but not _only_ the classic style.)

Reply:
I might be misreading this comment, but it seems like it's not that `classic` is a confusing name, but that the docstring here is not correct.

According to the discussion above. The docstring should read (if I understand correctly):

`*'classic'*, being the default, ensures built-in markers maintain the size they have always had, and for custom markers it makes sure the marker path is normalized to fit within a unit-square by affine scaling.`

This is *exactly* the classic style, as far as I can tell.

Reply:
Hence the sentence "Only applies to custom paths...". So this is definitely correct.

I'm happy to use a name other than "classic", but since I find that name fits pretty well, I would need input with alternative ideas. I guess this is also secondary compared to naming the argument itself - for which there is also no consensus yet it seems.

Reply:
To be clear, I am 100% behind the "classic" name. I think it is consistently applied and fits well. 

I am also very happy with "normalization".

However, wasn't one of the main benefits (and the impetus for this PR being revisited) that adding a "normalization" kwarg would allow us to address #15703? If so, I think that stating in the initial docstring that it only applies to custom paths (maybe just add "for now" if you don't like my more verbose wording).

Reply:
I'm cautious to document any possible future behaviour. (We had some new normalization or the toolmanager being anounced as "coming soon" for years, which doesn't really help people using the library in the present.)  
While it's important to think ahead and make sure to not block any roads (which is why we're having this discussion now), what's documented is the *actual* feature.

Reply:
100% onboard with not documenting "possible" future behavior. On the other hand, a wording closer to mine doesn't close the door to this feature (which is already implemented in my "paths" PR stack).

Mostly just being selfish because I have a strong feeling that if the current phrasing of the docstring gets merged, I'll get pushback when I try to change it in my own PR, whereas an equally specific and correct version could be written that doesn't specify that this keyword "only applies to custom paths".